### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by PicasoTheDeal**

### CODE REVIEW

**Review by AXER-doc — CHANGES_REQUESTED** [review]
One line of the code is NOT supported in C++11.
```
$ g++ -std=c++11 -pedantic -Werror -Wall 'The Game/main.cpp'
The Game/main.cpp:77:14: error: structured bindings only available with ‘-std=c++17’ or ‘-std=gnu++17’ [-Werror=c++17-extensions]
   77 |         auto [corNum, corPos] = checkGuess(randomNumber, userGuess);
      |              ^
```

**Comment by PicasoTheDeal** [comment]
You are right about that. what about we include other languages such as PHP, JavaScript, and C#.

**Review by PicasoTheDeal — COMMENTED** [review]
Approved

**Review by PicasoTheDeal — COMMENTED** [review]
I think we should add this as kata on code wars

File: README.md [hunk 1]
```diff
@@ -1 +1,42 @@
-# Guess-The-Number-Game
\ No newline at end of file
+# Number Guessing Game in C++
```
Comment (PicasoTheDeal) [inline README.md:1]: should we add this to leetcode

**Comment by PicasoTheDeal** [comment]
> One line of the code is NOT supported in C++11.
> 
> ```
> $ g++ -std=c++11 -pedantic -Werror -Wall 'The Game/main.cpp'
> The Game/main.cpp:77:14: error: structured bindings only available with ‘-std=c++17’ or ‘-std=gnu++17’ [-Werror=c++17-extensions]
>    77 |         auto [corNum, corPos] = checkGuess(randomNumber, userGuess);
>       |              ^
> ```

I debugged and it worked fine

**Pull request closed without merging**